pkg/controller/component/component_controller.go

```diff
@@ -104,8 +104,8 @@ type ReconcileComponent struct {
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileComponent) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	// Fetch the Component instance
-	instance := &devconsoleapi.Component{}
-	err := r.client.Get(context.TODO(), request.NamespacedName, instance)
+	cp := &devconsoleapi.Component{}
```

Comment:
What are your thoughts on using `component` instead of `cp` 
- doesn't collide with any other identifier. 
- `cp` has a different semantic meaning as well. 'Copy'

Reply:
I like `cp` as it is the short name of our component resource.

An interesting discussion on naming with idiomatic Go
>  "use short variable names over descriptive ones"

from http://wordaligned.org/articles/go-for-short-variable-names